app/controllers/api/membership_requests_controller.rb

```diff
@@ -0,0 +1,26 @@
+class API::MembershipRequestsController < API::RestfulController
+
+  def pending
+    load_and_authorize :group
```

Comment:
@HSalmon - another note! - I think we should be checking if the user can manage_membership_requests for this group before revealing pending and previous requests.